pkg/authorization/authorizer/scope/converter.go

```diff
@@ -130,43 +140,54 @@ func (clusterRoleEvaluator) Handles(scope string) bool {
 }
 
 func (e clusterRoleEvaluator) Validate(scope string) error {
-	_, _, err := e.getRoleNamespace(scope)
+	_, _, _, err := e.parseScope(scope)
 	return err
 }
 
-func (e clusterRoleEvaluator) getRoleNamespace(scope string) (string, string, error) {
+func (e clusterRoleEvaluator) parseScope(scope string) (string /*role name*/, string /*namespace*/, bool /*escalating*/, error) {
```

Comment:
Then just name the return values but don't change your return statements?

Or, just GoDoc and forgo the comments?
```go
// parseScope parses the requested scope, determining the requested role name, namespace, and if
// access to escalating objects is required
func (e clusterRoleEvaluator) parseScope(scope string) (string, string, bool, error) {
```